### PR TITLE
make go oneof work

### PR DESCRIPTION
**Pull request by pongad**
This is a follow up to https://github.com/googleapis/toolkit/pull/1016.
For the bug description, please see the linked PR.

@jba For the baseline change, please see the last commit.

### CODE REVIEW

File: src/main/java/com/google/api/codegen/viewmodel/OneofConfigView.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.api.codegen.viewmodel;
+
+import com.google.auto.value.AutoValue;
+
+@AutoValue
+public abstract class OneofConfigView {
+  // For illustration, assume we have this proto definition:
```
Comment (garrettjonesgoogle) [inline src/main/java/com/google/api/codegen/viewmodel/OneofConfigView.java]: This is an incomplete thought...

File: src/main/java/com/google/api/codegen/viewmodel/OneofConfigView.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  // message MyMessage {
+  //   oneof oneof_group {
+  //     string foo = 0;
+  //     int32 bar = 1;
+  //   }
+  // }
+
+  // The "oneof_group" in the example.
```
Comment (garrettjonesgoogle) [inline src/main/java/com/google/api/codegen/viewmodel/OneofConfigView.java]: why not proper javadocs?

**Comment by pongad** [comment]
@garrettjonesgoogle PTAL

**Comment by garrettjonesgoogle** [comment]
LGTM

**Pull request merged**